README.md

```diff
@@ -74,7 +75,8 @@ Consumer
 * `CA_CRT` - the certificate of the CA which signed the brokers' TLS certificates, for adding to the client's trust store
 * `USER_CRT` - the user's certificate
 * `USER_KEY` - the user's private key
-* `LOG_LEVEL` - logging level  
+* `LOG_LEVEL` - logging level
+* `HEADERS` = custom headers for receiving messages  
```

Comment:
tbh I don't understand what does HEADERS means for consumer. The consumer receives headers, it doesn't specify any key-value pair as for the producer that set headers when sending messages.

Reply:
Yeah sorry, it's just a draft, I wanted to try that.

Reply:
And when I was going through the code I saw that we don't need to specify that :) thanks